File: CSC205-Java/Module01-Java-Review-Software-Engineering-and-UML/debug01/Student.java
// A student enrolled in a course.
public class Student {
    public String name;
    private int id;

    public Student(String name, int id) {
        this.name = name;
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public String toString() {
        return name + " (" + id + ")";
    }
}

File: CSC101-Python/Module11-Classes-and-Objects/debug02/student.py
# student.py
# Tracks which courses each student is enrolled in.


class Student:
    courses = []

    def __init__(self, name):
        self.name = name

    def enroll(self, course):
        self.courses.append(course)

    def course_count(self):
        return len(self.courses)

    def __str__(self):
        return f"{self.name} is enrolled in {self.course_count()} course(s): {self.courses}"


ana = Student("Ana")
ben = Student("Ben")

ana.enroll("CSC101")
ana.enroll("MAT151")
ben.enroll("ENG101")

print(ana)
print(ben)
